test/controllers/houseConsignment/index/GrossWeightControllerSpec.scala

```diff
@@ -98,7 +98,7 @@ class GrossWeightControllerSpec extends SpecBase with AppWithDefaultMockFixtures
       val result = route(app, request).value
 
       status(result) mustEqual SEE_OTHER
-      redirectLocation(result).value mustEqual onwardRoute.url
+      redirectLocation(result).value mustEqual "/manage-transit-movements/unloading/AB123/transit-unloading-permission-discrepancies/house-consignment/1"
```

Comment:
Needing to change these is a sign that you aren't adequately mocking the navigator. If you look in `AppWithDefaultMockFixtures` you'll see a bunch of fake navigators. You'll need to add ones for your new navigator classes